django/thunderstore/frontend/tests/data/scripts.html

```diff
@@ -0,0 +1,310 @@
+
```

Comment:
Are the odd empty lines and indentations intentional for testing's sake?

Reply:
Yeah, this file is just copypasted output from what the site rendered so I didn't bother formatting it. I felt it's the most accurate representation of what the functions are used for in reality.